docs/migration/dual-engine-testing.md

```
@@ -0,0 +1,340 @@
+
+```bash
+# Run with performance profile
+mvn test -pl judo-tatami-psm2asm -Dtest=RealisticPerformanceTest -Pperformance
+
+# Run all performance tests
+mvn test -Pperformance
+```
+
+## Test Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)